keyboards/maartenwut/eon40/config.h

```diff
@@ -0,0 +1,254 @@
+/* Copyright 2019 Maarten Dekkers <maartenwut@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4705
+#define PRODUCT_ID      0x0140
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Maartenwut
+#define PRODUCT         Eon40
+#define DESCRIPTION     A 40% ortholinear PCB
+
+/* key matrix size */
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 13
```

Comment:
You only have 12 pins defined for `MATRIX_COL_PINS`.

```suggestion
#define MATRIX_COLS 12
```

Reply:
Funny that it built though